### PR TITLE
[MM-21870] Default help text for "Invite People” should not wrap to another line

### PR DESCRIPTION
**Pull request by bradjcoughlin**
#### Summary
Expands the minimum width of the Main Menu dropdown to prevent English help text from wrapping to a second line.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-21870

#### Related Pull Requests
N/A

#### Screenshots
Screen Shot 2020-01-27 at 2 57 35 PM
Screen Shot 2020-01-27 at 2 57 54 PM

### CODE REVIEW

**Comment by lindy65** [comment]
Hi @bradjcoughlin - tested on the spinwick server and am still seeing the help text wrap to a second line:

image

**Comment by esethna** [comment]
@asaadmahmood can you help with providing some direction on the padding? I agree it feels quite spacey.

**Comment by asaadmahmood** [comment]
@lindy65 Should be fixed now after my changes.
@esethna Adjusted the spacing. Some menu items also were inconsistent in their spacing, fixed that too.

Screenshot 2020-01-30 at 6 14 19 AM

**Comment by bradjcoughlin** [comment]
Do we still need UX to review then, or does @asaadmahmood 's approval cover that? Also, I'm not sure what was inconsistent, exactly. Inconsistent visually or programmatically?

**Comment by asaadmahmood** [comment]
@bradjcoughlin Not specific to any of your changes, but both visually and progamatically.
The class `MenuItem__primary-text` was not wrapping the text in a menu item link, but were doing so in a menu item button. That was causing the button to be bigger in terms of height than the links.

And I think Mike is taking a week off, so we don't have to wait for him.

**Comment by bradjcoughlin** [comment]
@asaadmahmood Got it. I had added the class to give greater control over the help text but it looks like I didn't add to the link. Thanks!

**Comment by amyblais** [comment]
@bradjcoughlin Is this ready to merge or does it need any other reviews?

**Comment by bradjcoughlin** [comment]
@amyblais this is ready to merge. Working on it now.

File: components/widgets/menu/menu_items/menu_item_link.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ type Props = {
     text: React.ReactNode;
 }
 
-export const MenuItemLinkImpl: React.FC<Props> = ({to, text}: Props) => <Link to={to}>{text}</Link>;
+export const MenuItemLinkImpl: React.FC<Props> = ({to, text}: Props) => <Link to={to}><span className='MenuItem__primary-text'>{text}</span></Link>;
```
Comment (hmhealey) [inline components/widgets/menu/menu_items/menu_item_link.tsx]: 0/5, but you could just pass the `className` into the `Link` directly to have it put on the `<a>` tag

Comment (bradjcoughlin) [reply components/widgets/menu/menu_items/menu_item_link.tsx]: That's cleaner, I'll make a small follow-up PR to improve this @hmhealey

**Pull request merged**